{"topicSections":[{"title":"Instance Properties","anchor":"Instance-Properties","identifiers":["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/localizedDescription"],"generated":true}],"metadata":{"modules":[{"name":"VigramSDK"}],"title":"Error Implementations","role":"collectionGroup","roleHeading":"API Collection"},"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError"]]},"variants":[{"paths":["\/documentation\/vigramsdk\/authenticationerror\/error-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"article","sections":[],"identifier":{"url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/Error-Implementations","interfaceLanguage":"swift"},"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"url":"\/documentation\/vigramsdk","kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","abstract":[],"title":"VigramSDK","images":[{"type":"icon","identifier":"logo.png"}],"type":"topic","role":"collection"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/localizedDescription":{"role":"symbol","url":"\/documentation\/vigramsdk\/authenticationerror\/localizeddescription","type":"topic","abstract":[],"kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/localizedDescription","title":"localizedDescription","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"localizedDescription"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"}]},"logo.png":{"identifier":"logo.png","variants":[{"url":"\/images\/com.vigram.VigramSDK\/logo.png","traits":["1x","light"]}],"alt":"Vigram logo.","type":"image"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError":{"abstract":[{"text":"AuthenticationError contains custom failure events when connecting to Authentication process.","type":"text"}],"title":"AuthenticationError","role":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError","kind":"symbol","type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"AuthenticationError","kind":"identifier"}],"navigatorTitle":[{"text":"AuthenticationError","kind":"identifier"}],"url":"\/documentation\/vigramsdk\/authenticationerror"}}}